utils/firestore.js

```diff
@@ -418,8 +418,8 @@ const retrieveParticipantsEligibleForIncentives = async (siteCode, roundType, is
 const removeParticipantsDataDestruction = async () => {
     try {
         let count = 0;
-        const millisecondsInTwoDays = 172800000; // 2days
-        const stubFieldArray = ['104278817', '119449326', '153713899', '173836415', '231676651', '262613359', '268665918', '269050420', '304438543', '359404406', '399159511', '407743866', '412000022', '471168198', '479278368', '524352591', '526455436', '544150384', '558435199', '577794331', '592227431', '613641698', '664453818', '744604255', '747006172', '765336427', '773707518', '826240317', '831041022', '883668444', '996038075', 'token', 'Connect_ID', 'query', 'pin', 'state', '371067537', '827220437', '699625233','436680969','736251808','480305327','795827569', '564964481', '454205108','919254129','982402227','821247024','914594314','635994596','297147359','241236037', '912301837','869588347','506826178', '153211406','521824358','703385619','634434746','892050548'];
+        const millisecondsInTwoDays = 5184000000; // 60days
```

Comment:
change the variable name to `millisecondsInSixtyDays` from `millisecondsInTwoDays`